tests/backup/backup_helper.go

```diff
@@ -6545,3 +6564,51 @@ func SuspendAndDeleteSchedule(backupScheduleName string, schedulePolicyName stri
 	}
 	return nil
 }
+
+// ValidateCustomResourceRestore validates restore taken of custom resource
+func ValidateCustomResourceRestore(ctx context.Context, orgID string, resourceTypesFilter []string, restoreNamespaceScMappingMap map[string][]map[string]string, restoreContextMap map[string][]*scheduler.Context) error {
```

Comment:
Split the map `restoreNamespaceScMappingMap` into `restoreNamespaceMap` and `restoreScMap`

Reply:
Change the name of the func to `ValidateCustomResourceRestores`

Reply:
done

Reply:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/3912/consoleFull